src/x509/clu_x509_sign.c

```diff
@@ -119,6 +140,17 @@ static void wolfCLU_CertSignSetRandFile(WOLFCLU_CERT_SIGN* csign, char* f)
     if (csign != NULL) {
         wolfSSL_BIO_free(csign->randFile);
         csign->randFile = wolfSSL_BIO_new_file(f, "wb+");
+        if (csign->randFile == NULL) {
+            WOLFCLU_LOG(WOLFCLU_E0, "Error reading rand file");
+        }
+        else {
+            byte seed[256];
+            int  seedSz = 256;
+
+            seedSz = wolfSSL_BIO_read(csign->randFile, seed, seedSz);
+            wolfSSL_RAND_add(seed, seedSz, 0);
```

Comment:
Interestingly this `wolfSSL_RAND_add` function doesn't really do anything... But if a rand add method was setup it will call it....